editor/src/main/java/net/rpgtoolkit/editor/ui/IntegerField.java

```diff
@@ -40,12 +40,12 @@ public static NumberFormatter getIntegerFormatter() {
   }
 
   @Override
-  public Long getValue() {
+  public Integer getValue() {

```

Comment:
If we're switching to using ints instead of longs, then the docstrings for IntegerField and its methods should be updated.